README.md

```diff
@@ -1,9 +1,13 @@
-# Chronicle Examples
+# Chronicle Bootstrap
 
-This project contains contributed examples for Chronicle.  The markdown
-documents in this repository are published on
-[their own site](https://examples.btp.works). Documentation for
-Chronicle in general may be found [here](https://docs.btp.works/chronicle/).
+This repo serves as a base for setting up your own Chronicle domain.
+It contains an example domain, and a Makefile which can be used to
```

Comment:
To me, it would be better to highlight that this codebase works essentially the same as the chronicle-examples repository and refer to the manufacturing and other examples there.  Rather than include `manufacturing`  here.  I can imagine the question, "why is my build running so slow"  because a user didn't delete the manufacturing example.  

At the very least move the manufacturing domain out into an examples directory or similar, and have the walkthrough have the user copy the domain in to minimize confusion.

Reply:
@scealiontach Thanks, this makes more sense, I've updated the readme

Reply:
OK looks good, but now "it does not contain an example domain" :grin:

Reply:
So close, I've updated it and added a link to the examples